README.md

```diff
@@ -132,6 +132,10 @@ selectively enabled or disabled:
   true if cljfmt should collapse consecutive blank lines. This will
   convert `(foo)\n\n\n(bar)` to `(foo)\n\n(bar)`. Defaults to true.
 
+* `:split-maps?` -
```

Comment:
Let's make this a little more descriptive. For example: `:split-keypairs-over-multiple-lines?`.